include/list.h

```diff
@@ -33,6 +33,7 @@
  * @member:     the name of the member within the struct.
  *
  */
+#undef container_of
 #define container_of(ptr, type, member) ({                \
```

Comment:
Where else is macro `container_of()` being introduced? Isn't it there defined as the same here or equivalent? If so, instead of the `undef`, we can drop the macro `container_of()` here; we may need to include the new header to make sure that the macro is available.

Reply:
It's defined here:

https://github.com/cjdoucette/dpdk/blob/gatekeeper/lib/librte_eal/common/include/rte_common.h

I would be OK dropping our definition of `container_of()`, it just seemed like to deal with the redefinition of `offsetof()` we did an `undef` and then `define` (above).

Reply:
Let's drop our definition. Don't forget to add `#include <rte_common.h>`.